pandas/core/base.py

```diff
@@ -942,7 +948,22 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
 
         See Also
         --------
-        numpy.ndarray.argmax
+        numpy.ndarray.argmax : Returns the indices of the maximum values along an axis.
```

Comment:
Do we have a pandas `argmin` that is worth adding here?

Reply:
```
See Also
--------
numpy.ndarray.argmax : Returns the indices of the maximum values along an axis.
pandas.Series.argmin : Return a ndarray of the minimum argument indexer.
```
Would this be sufficient?

Reply:
Looks good. I'm not sure if the `pandas.` prefix is needed, may be you can check other docstrings and see.

And for the descriptions, it could make sense to say something like "Equivalent method for numpy arrays.", "Same, but returning the minimum". I think for users could be easier to understand what those methods do.

Reply:
Yes, I've noticed through `scripts/validate_docstrings.py` that I could safely omit `pandas.` to match the guidelines.

I agree with your proposed descriptions, as current description is redundant with each function description